packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props

```diff
@@ -15,18 +15,26 @@
     using the SDK style projects. For e.g., see https://github.com/microsoft/msbuild/issues/1333
     
     Irrespective of whether '$(TargetFrameworkIdentifier)' is '.NETCoreApp' or '.NETFramework', 
-    the minimum value of $(_TargetFrameworkVersionWithoutV) we will be testing for is '3.0'
+    the minimum value of $(_TargetFrameworkVersionValue) we will be testing for is '3.0'
     
-    Note:
-      Please see https://github.com/microsoft/msbuild/issues/3212 for a discussion about the use of 
-      the private $(_TargetFrameworkVersionWithoutV) property - which will likely remain supported and
-      is safe to use here. 
   -->
     <_WindowsDesktopSdkTargetFrameworkVersionFloor>3.0</_WindowsDesktopSdkTargetFrameworkVersionFloor>
+
+    <!-- 
+      Represents an undefined TFV value. This will be used in comparisons of _TargetFrameworkVersionValue (defined in Microsoft.NET.WindowsDesktop.targets)
+      to identify when a TFV is undefined
+    -->
+    <_UndefinedTargetFrameworkVersion>0.0</_UndefinedTargetFrameworkVersion>
+    
+    <!--
+      Initial/Default value set to 'undefined'. Updated in Microsoft.NET.WindowsDesktop.targets
+    --> 
+    <_TargetFrameworkVersionValue>$(_UndefinedTargetFrameworkVersion)</_TargetFrameworkVersionValue>
```

Comment:
Is this needed? Are there circumstances where this file will be imported but `Microsoft.NET.WindowsDesktop.targets` won't?

Reply:
Clearly we support such a pattern, like this `<Import Sdk="Microsoft.NET.Sdk.WindowsDesktop" Project="Sdk.props" />`. Whether someones custom-build setup will actually do something like that is anybody's guess - I figured we should be defensive.

Reply:
I wouldn't bother supporting it personally. Lots of things break if you don't have props/targets in matched pairs (despite the failed "common.targets imports common.props if it hasn't already been" attempt). But no strong objection.